webapp/templates/partials/scoreboard_table.html.twig

```diff
@@ -119,11 +120,12 @@
         {# process medal color #}
         {% set medalColor = '' %}
         {% if showLegends and medalsEnabled and score.team.category in medalCategories and score.numPoints > 0 %}
-            {% if score.rank <= contest.goldMedals %}
+            {% set medalCount = medalCount + 1 %}
```

Comment:
Should this line not also be added to two elseifs below?

Reply:
In the following gold, silver and bronze ifelse(line 122/124/126) is need to be added, so I mentioned this line of code to the front. I have tested this code and it works.

Reply:
Its logic is that as long as the award conditions are met, add 1 to the `medalCount`, and then determine which award should be put.